LabDrivers/utils.py

```diff
@@ -280,7 +421,7 @@ def list_drivers(interface=[INTF_VISA, INTF_PROLOGIX, INTF_SERIAL, INTF_NONE]):
     for file_name in driver_files:
 
         if file_name.endswith('.py') \
-                and (not file_name == 'Tool.py' and not file_name == '__init__.py' and not file_name == 'utils.py'):
```

Comment:
this could go in the function `get_drivers_list()` for example

Reply:
beat you to it :P however I have not removed the if statement condition